build-tests/x86/centos/test-image-live-disk-v9/appliance.kiwi

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- The line below is required in order to use the multibuild OBS features -->
+<!-- OBS-Profiles: @BUILD_FLAVOR@ -->
+
+<image schemaversion="7.4" name="kiwi-test-image-live-disk-v9">
+    <description type="system">
+        <author>Marcus Schaefer</author>
+        <contact>ms@suse.de</contact>
```

Comment:
It's not a _huge_ deal, but you should probably fix your email address here. 😉

Reply:
Oh well thanks, I'm still so used to it ;)

Reply:
done